code/script.js

```diff
@@ -1,9 +1,15 @@
 // A function that adds and remove the class "active" on the section you click on.
 // We haven't really talked about ´this´ yet, but we will... ;)
 // console.log(this) to see how it works
-function toggle() {
-  this.classList.toggle("active")
-}
 
-// Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+// Find all elements with the class "accordion-item-header" and assign them to the variable "accordionItemHeader".
+  const accordionItemHeaders = document.querySelectorAll(".accordion-item-header");
+
+// Put all accordionItemHeader in the accordionItemHeaders ListeningStateChangedEvent, and toggle the class active depending on whether they already are active or not 
+    accordionItemHeaders.forEach(accordionItemHeader => {
+    accordionItemHeader.addEventListener("click", event => {
+    accordionItemHeader.classList.toggle("active");
+  });
+});
```

Comment:
I am really like your JavaScript code! I think it is the best and shortest solution for accordion :) Good job!